src/handlebars/support.handlebars

```diff
@@ -59,6 +59,360 @@
         and we will work with you and the appropriate development team to resolve it.
         Any fixes we identify will be delivered as part of the next AdoptOpenJDK release.
       </p>
+	  
+		<h3 id="openjdk8_platforms">OpenJDK 8</h3>
```

Comment:
As another general comment, I wonder if OpenJDK 8 and 11 could be represented in the same table..